scripts/joinmarket-qt.py

```diff
@@ -649,6 +651,27 @@ def qt_directsend_callback(rtxd, rtxid, confs):
         self.spendstate.updateRun('running')
         self.startJoin()
 
+    def getMaxCJFees(self, relfee, absfee):
+        """ Used as a callback to decide relative and absolute
+        maximum fees for coinjoins, in cases where the user has not
+        set these values in the config (which is the default)."""
+        if relfee is None:
+            relfee = get_default_max_relative_fee()
+        if absfee is None:
+            absfee = get_default_max_absolute_fee()
+        msg = ("Your maximum absolute fee in from one counterparty has been "
+              "set to: " + str(absfee) + " satoshis.\n"
```

Comment:
Use `btc.amount_to_str()` here to display amount both in BTC and sats?

Reply:
Thanks. Makes sense.

Reply:
Overwhelmed with other stuff, so deciding not to do this right now, since it's only presentation-layer and what we have here is correct. If I changed it would need to go through one more testing round.
You or I should open an issue/PR on this for the future. Thanks again.